torch/csrc/autograd/profiler.h

```diff
@@ -280,8 +280,8 @@ struct TORCH_API Event final {
   // signed to allow for negative intervals, initialized for safety.
   int64_t cpu_ns_ = 0;
   at::StringView name_;
-  EventKind kind_;
-  uint16_t thread_id_;
+  EventKind kind_ = EventKind::Mark;
```

Comment:
This could lead to really hard to reason about bugs, i.e. an extra Mark event may result in the profiler not being able to match events when parsing the CPU trace in Python, and it'll be hard to find that the error actually comes from here. 

Though, I guess not having a default value is prone to the same errors. We should never call the `Event` constructor without a kind_ argument anyways.